pfcpiface/interface.go

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright(c) 2020 Intel Corporation
+
+package main
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"net"
+)
+
+type fastPath interface {
+	exit()
+	setUpfInfo(u *upf, conf *Conf)
+	setInfo(udpConn *net.UDPConn, updAddr net.Addr, pconn *PFCPConn)
+	sim(u *upf, method string)
+	sendMsgToUPF(method string, pdrs []pdr, fars []far) uint8
+	sendDeleteAllSessionsMsgtoUPF()
+	summaryLatencyJitter(uc *upfCollector, ch chan<- prometheus.Metric)
+	portStats(uc *upfCollector, ch chan<- prometheus.Metric)
```

Comment:
will P4RT also implement this? if so we should move this to a `monitoring` interface probably?

Reply:
this would be needed in p4rt implementation also. Not sure why we need a separate interface for this. the functions basically query the underlying fast path for information. so can be implemented in the same interface I think.

Reply:
This is something we can work on later when we get monitoring support in p4rt ?

Reply:
yes we can add the support later